heft-plugins/heft-storybook-plugin/src/webpack-dev-middleware-workaround.d.ts

```diff
@@ -0,0 +1,9 @@
+// The webpack-dev-middleware@5.3.1 project has a phantom dependency on @types/node
+// and so "npm install" chooses the "*" latest version.  As a result, their index.d.ts
+// has a reference to fs.StatSyncFn which is new in @types/node@16.  As of this writing
+// Node 12 is still LTS so we need to support it.
+// Upstream issue: https://github.com/webpack/webpack-dev-middleware/issues/1194
```

Comment:
Looks like this fix was published in version 5.3.2: https://github.com/webpack/webpack-dev-middleware/releases/tag/v5.3.2

Reply:
Oh nice! Good catch.